Week1/assignments/usmaila-abdoul/Anagrams.js

```diff
@@ -0,0 +1,43 @@
+// 1 split string to subStrings
+// 2 sort all subStrings and compare each substirng with all the other substrings to find duplicate
+
+function splitStrToSubstrs(string) {
+    let subStr = [];
+
+    for (let i = 0; i < string.length; i++) {
+        for (let j = i + 1; j < string.length + 1; j++) {
+            subStr.push(string.slice(i, j))
+        }
+    }
+    return subStr;
+}
+
+function Anagrams(string) {
+    var i, j, anagrams = [];
```

Comment:
Try to avoid using `var` now that you can use `let` and `const`
You can read more about why [here](https://scotch.io/tutorials/understanding-hoisting-in-javascript)